spot-controller/App.js

```diff
@@ -26,6 +27,7 @@ export default class App extends React.Component {
         super(props);
 
         this.state = {
+            loading: __DEV__,
```

Comment:
what does it do ? why set to `__DEV__` ?

Reply:
Right now only dev does something async (look in local storage). Whereas if it's not dev then it skips looking in local storage. So on dev what happens is the code entry screen displays, find a url or does not, and then displays the webview. But while it's reading from local storage, it shows the code entry screen.

In a follow up commit where I have to store if the app has ever been launched this.state.loading gets started as true, not __DEV__, and will be set to false after async storage has been searched for some flag saying the app has been launched before.

Reply:
My understanding of the loading screen purpose was to show it during the time when the WebView is loading the page

Reply:
So you want me to remove hiding the flash of the code entry screen. I don't mind.

Reply:
I don't know if that's what you want but I went ahead and did it.